app/assets/stylesheets/presences.scss

```diff
@@ -75,11 +79,11 @@
 }
 
 .Presences-user--available {
-  background-color: green;
+  background-color: #00a878;
 }
 
 .Presences-user--service {
-  background-color: darkred;
+  background-color: #f03434;
```

Comment:
Don't we want to use color variables here?

Reply:
I told you to ignore the changes in the Presences for now.

This is a start PR... 😛  Can we please move forward?

Reply:
@luistorres approved, merge when you want